doc/api/util.md

```diff
@@ -620,8 +620,35 @@ util.inspect(obj);
 added: v6.6.0
 -->
 
-* {symbol} that can be used to declare custom inspect functions, see
-[Custom inspection functions on Objects][].
+* {symbol} that can be used to declare custom inspect functions.
+
+This is a shared symbol, [`Symbol.for('util.inspect.custom')`], which is registered
+in the [global symbol registry](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for).
```

Comment:
Would you be so kind and move the actual link to the bottom of the file? :-) That allows the text to still stay readable.

Reply:
[Done](https://github.com/nodejs/node/pull/20857/commits/946afaf2623be4e4cfcd583abc786bb2bd0a95e3).